### ISSUE TITLE
false positive `partial-no-import` 

### ISSUE THREAD

**Issue by evilebottnawi** [kristerkari/stylelint-scss#69]
Example
```scss
@import "_bootstrap/variables";
@import "_font-awesome/variables";
```

**Comment by dryoma** [comment]
What is the name of the file that has those `@import`s?

**Comment by evilebottnawi** [comment]
@dryoma `_variables.scss`

**Comment by dryoma** [comment]
Well, that's no false positive. What this rule prohibits is not importing some partials (that would be strange), but using imports *inside partials*.

**Comment by evilebottnawi** [comment]
@dryoma i think it is false positive, because if i remove `_` from `_bootstrap` name directory, rule works fine.

**Comment by dryoma** [comment]
Sorry, couldn't look into this for a while.

So I tested these scenarios:

file: `_file.scss`, code: 
```scss
@import "_bootstrap/variables";
@import "_font-awesome/variables";
```
2 warnings, 'cause `_file.scss` is a partial and filenames from `@improt`s don't matter.

file: `_file.scss`, code: 
```scss
@import "bootstrap/variables";
@import "font-awesome/variables";
```
2 warnings, 'cause `_file.scss` is a partial and filenames from `@improt`s don't matter.

file: `dir/_file.scss`, code: 
```scss
@import "bootstrap/variables";
@import "font-awesome/variables";
```
2 warnings, 'cause `dir/_file.scss` is a partial and filenames from `@improt`s don't matter.

file: `_dir/file.scss`, code: 
```scss
@import "bootstrap/variables";
@import "font-awesome/variables";
```
0 warnings, 'cause `_dir/file.scss` is not a partial.

All the results I got are as expected. So I'm really confused what went wrong in your case.

**Comment by evilebottnawi** [comment]
@dryoma yes, you are right, inattentively I read the rule and expect another behavior :sob:

**Comment by dryoma** [comment]
I see :) BTW if you want to control the names of imported files, you might want to check out at-import-[`no-partial-leading-underscore`](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/at-import-no-partial-leading-underscore/README.md), [`at-import-partial-extension-blacklist`](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/at-import-partial-extension-blacklist/README.md) and [`at-import-partial-extension-whitelist`](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/at-import-partial-extension-whitelist/README.md).

**Comment by evilebottnawi** [comment]
@dryoma thanks :smile:
